### PR TITLE
Change method arg alignment

### LINKED ISSUE #3712: method_argument_space with multiline arguments
#### The PHP version you are using (`$ php -v`):
=> 7.2.4

#### PHP CS Fixer version you are using (`$ php-cs-fixer -V`):
=> 2.11.1

#### The command you use to run PHP CS Fixer:

```php
<?php
function a($a, $b,
           $c, $d) {
}
```
```shell
php-cs-fixer fix sandbox.php --rules=@PSR2 --dry-run --diff
```
```diff
--- Original
+++ New
@@ @@
 <?php

-function a($a, $b,
-           $c, $d) {
+function a(
+    $a,
+    $b,
+           $c,
+    $d
+) {
 }
```

Extra spaces before `$c` could be removed. Perhaps this isn't `MethodArgumentSpaceFixer`'s responsibility, or it could have a new option `ensure_aligned` if `ensure_fully_multiline` is `true`.

#### If applicable, please provide minimum samples of PHP code (as plain text, not screenshots):

Here's a test case with expected output:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/compare/master...nobuf:MethodArgumentSpaceFixer-with-multiline-args

### PR DESCRIPTION
**Pull request by Billz95**
Fixes #3712 
As mentioned in the above issue, the code will not be aligned properly when forced into multiline (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3712#issue-316138499)
```php
<?php
function a($a, $b,
           $c, $d) {
}
```
would be fixed to
```diff
--- Original
+++ New
@@ @@
 <?php

-function a($a, $b,
-           $c, $d) {
+function a(
+    $a,
+    $b,
+           $c,
+    $d
+) {
 }
```
However, the expected result is 
```diff
--- Original
+++ New
@@ @@
 <?php

-function a($a, $b,
-           $c, $d) {
+function a(
+    $a,
+    $b,
+    $c,
+    $d
+) {
 }
```
This is caused by function `MethodArgumentSpaceFixer::fixNewline`, where it simply returns without applying any fix when the token after `,` is newline.

This is not ideal, from the above code snippet, we can see the token after `$b,` is `newline + many spaces`. Instead, what we expect should be `newline + indentation`.

Therefore, by simply removing `$this->isNewline($token[$index + 1])` from early return check, it fixes this indentation issue.

Any related unit tests has also been modified to allow proper alignment.

However, I'm not sure whether this will have any other side effect, if so, please point it out. Thanks!

### CODE REVIEW

File: tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php [hunk 1]
```diff
@@ -718,7 +717,7 @@ class FooClass
     function ($arguments) {
     echo 'a', 'b';
 },
```
Comment (dmvdbrugge) [inline tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php]: I'd say this line should've been indented as well, but that might be out-of-scope for this issue/pr.

**Comment by Billz95** [comment]
> Although one thing - let's target `2.12`branch.

Thx for suggestion, should I update this PR to patch `2.12` branch, or open a new one? 
Also, what's the rationale for targeting `2.12` rather than the latest `2.14`?

**Comment by dmvdbrugge** [comment]
Yes you should update this PR to target 2.12

The rationale is to always target the lowest possible maintained branch for bugfixes. 2.12 and 2.13 still get (bugfix) releases.

Once merged, your change will be merged up into 2.13, 2.14, master, and 3.0 by the mergers.

That said, 2.12 is a longterm support branch, I don't know how long 2.13 (and 2.14 once 2.15 is out) will be supported.

**Comment by Billz95** [comment]
> Yes you should update this PR to target 2.12
> 
> The rationale is to always target the lowest possible maintained branch for bugfixes. 2.12 and 2.13 still get (bugfix) releases.
> 
> Once merged, your change will be merged up into 2.13, 2.14, master, and 3.0 by the mergers.
> 
> That said, 2.12 is a longterm support branch, I don't know how long 2.13 (and 2.14 once 2.15 is out) will be supported.

Thanks, that's really informative. 
I've opened a new PR at #4280 as I don't know how to change the base branch on git PR..

**Comment by Billz95** [comment]
Pull Request now targets ver `2.12`, see #4280

**Comment by kubawerlos** [comment]
@Billz95 

> I don't know how to change the base branch on git PR.

When you click "Edit" on PR you have possibility to change the branch.

**Comment by Billz95** [comment]
> @Billz95
> 
> > I don't know how to change the base branch on git PR.
> 
> When you click "Edit" on PR you have possibility to change the branch.

Amazing! Will use this feature next time! 😄

**Pull request closed without merging**